src/core/lib/experiments/experiments.h

```diff
@@ -40,14 +40,15 @@ inline bool IsEventEngineClientEnabled() { return IsExperimentEnabled(7); }
 inline bool IsMonitoringExperimentEnabled() { return IsExperimentEnabled(8); }
 inline bool IsPromiseBasedClientCallEnabled() { return IsExperimentEnabled(9); }
 inline bool IsFreeLargeAllocatorEnabled() { return IsExperimentEnabled(10); }
+inline bool IsEventEngineServerEnabled() { return IsExperimentEnabled(11); }
```

Comment:
nit: suggest `IsEventEngineListenerEnabled`, for consistency. `s/Server/Listener/g` when referring to EventEngine objects.